recipes/nsync/all/conanfile.py

```diff
@@ -0,0 +1,98 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class NsyncConan(ConanFile):
+    name = "nsync"
+    homepage = "https://github.com/google/nsync"
+    description = "Library that exports various synchronization primitive"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("c", "thread", "multithreading", "google")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False], "shared": [True, False]}
+    default_options = {"fPIC": True, "shared": False}
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True,
+                  destination=self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["NSYNC_ENABLE_TESTS"] = False
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "CMakeLists.txt"),
+            "set (CMAKE_POSITION_INDEPENDENT_CODE ON)", "")
+
+        if self.settings.os == "Windows" and self.options.shared:
+            ar_dest = \
+                "ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} " \
+                "COMPONENT Development"
+            rt_dest = 'RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"'
```

Comment:
I'd apply this change regardless of the OS and build type, even if only makes sense there.
Also keep in mind you're removing the `COMPONENT`, I don't know the implications of this in this particular case.

Reply:
Per comment in https://cmake.org/cmake/help/latest/command/install.html:

> **`COMPONENT`**
> Specify an installation component name with which the install rule is associated, such as "runtime" or "development". **_During component-specific installation_** only install rules associated with the given component name will be executed. During a full installation all components are installed unless marked with `EXCLUDE_FROM_ALL`. If `COMPONENT` is not provided a default component "Unspecified" is created. The default component name may be controlled with the `CMAKE_INSTALL_DEFAULT_COMPONENT_NAME` variable.

Because we don't perform **component-specific installation** probably this is not the case for us.

As another solution, I think we can safely add `COMPONENT RuntimeLibraries` which is also looks like custom one